extra-utils/peazip/autobuild/build

```diff
@@ -0,0 +1,52 @@
+if   [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
```

Comment:
Did you reference this build script from anywhere? If so, please indicate the origin(s).